config/routes.rb

```diff
@@ -1,76 +1,17 @@
 Rails.application.routes.draw do
 
-  scope "asset-directory" do
+  scope path: "asset-directory" do
     resources :assets, as: 'assets', path: 'all'
     get 'assets/laptops', to: 'assets#laptops', as: 'laptops', path: 'laptops'
```

Comment:
Made some mistakes so undid all changes and started over and forgot to take that away